chatlogger.lua

```diff
@@ -8,13 +8,22 @@
 --]]
 
 function OnStartup()
-	tConfig, tChatHistory = {
+	tConfig, tChatHistory, tTopics = {
 		sBotName = SetMan.GetString( 21 ) or "PtokaX",
 		sProfiles = "012",		-- No history for commands from users with profiles
 		sLogsPath = "/www/ChatLogs/",
 		sTimeFormat = "[%I:%M:%S %p] ",
+		sPath = Core.GetPtokaXPath().."scripts/texts/",
+		sTickersList = "tickers.txt",
 		iMaxLines = 100,
-	}, { "Hi!" }
+		iTickerDelay = 6 * 60 * 60 * 10^3,		-- 6 hours to milliseconds
+		iTickerID = false,
+		iTopicIndex = 0,
+	}, { "Hi!" }, {}
+	for sLine in io.lines( tConfig.sPath..tConfig.sTickersList ) do
```

Comment:
I thought [`io.lines`][1] would do that by itself.

Apparently, it raises an error instead.

  [1]: http://lua.org/manual/5.2/manual.html#pdf-io.lines